transformer/README.md

```diff
@@ -0,0 +1,87 @@
+# Introduction
+
+GPSS 1.9.0 introduces a "Transformer" type plugin for Kafka and RabbitMQ job. Transformer gives customer a chance to inject some custom logic right after messages were read from Kafka/RabbitMQ and before they were sent to downstream. We'll give an example to show how to write your own transformer plugin in the following sections. The source code of example is in [examples](./examples) directory.
+
+# How to write and use your own transformer
+
+In this section, we'll use an example to show how to write your own transformer plugin. Our example transformer is [simple_transformer.go](./examples/simple_transformer.go). The implementation is really simple, it takes json data like `{"A":2,"B":1}` as input and takes an operator from properties.If operator=="+", it puts C=A+B back, if operator=='-', it puts C=A-B back.
+
+## 1 Implement `init` and `transform` function
+
+There are two main functions to be implemented. We call them `init` function and `transform` function,  this doesn't mean the function must have name "init" or "transform", you can name your function according to your needs.
+
+###  1.1 init function
+
+Init function will be invoked when the transformer plugin is first loaded. It's guaranteed to be invoked only once even if the plugin is used in multi jobs.
```

Comment:
Add the description that Init function is not mandatory

Reply:
> Add the description that Init function is not mandatory

fixed